lib/appsignal/plug.ex

```diff
@@ -167,7 +169,8 @@ if Appsignal.plug?() do
     )
 
     def extract_request_headers(%Plug.Conn{req_headers: req_headers}) do
-      for {key, value} <- req_headers, key in @header_keys do
+      for {key, value} <- req_headers,
+          key in (Config.request_headers() || @fallback_request_headers) do
```

Comment:
can we print a warning here when the fallback is being used?
And link to some documentation where we explain the new setup.

Reply:
We'll add those in part 2, when we start writing default request header configurations. Right now, this patch just adds the option to override the defaults.